### PR TITLE
Move multizone e2e to e2e/scheduling from e2e/multicluster 

### PR DESCRIPTION
**Pull request by marun**
afaik these tests should be the responsibility of @kubernetes/sig-scheduling-pr-reviews instead of @kubernetes/sig-multicluster-pr-reviews. 

```release-note
NONE
```

### CODE REVIEW

**Comment by nikhiljindal** [comment]
cc @justinsb

**Comment by timothysc** [comment]
Let's chat about this one tomorrow.

**Comment by timothysc** [comment]
In looking at these tests, they are actually under @kubernetes/sig-scheduling-misc 's perview.   I'd move into those buckets.

File: test/e2e/scheduling/ubernetes_lite_volumes.go [hunk 1]
```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package multicluster
+package scheduling
```
Comment (timothysc) [inline test/e2e/scheduling/ubernetes_lite_volumes.go]: I'd change this file to use SIGDescribe and remove the [sig-storage] bit then lgtm

**Review by timothysc — CHANGES_REQUESTED** [review]

File: test/e2e/scheduling/ubernetes_lite_volumes.go (hunk 1 above)
Comment (marun) [reply test/e2e/scheduling/ubernetes_lite_volumes.go]: Done.

**Review by timothysc — APPROVED** [review]
/lgtm 
/approve 

Still need to determine why the other tests are failing though.

**Comment by marun** [comment]
Rebased on reflex, apparently I haven't internalized that this isn't necessary.  Still, pagan rituals are par for the course when it comes to dealing with test failures that could not possibly be triggered by this PR.

File: test/e2e/scheduling/ubernetes_lite_volumes.go [hunk 2]
```diff
@@ -32,7 +32,7 @@ import (
 	"k8s.io/kubernetes/test/e2e/framework"
 )
 
-var _ = Describe("[sig-storage] Multi-AZ Clusters", func() {
+var _ = SIGDescribe("Multi-AZ Cluster Volumes", func() {
```
Comment (bsalamat) [inline test/e2e/scheduling/ubernetes_lite_volumes.go]: Why is this test related to scheduling?

**Review by bsalamat — CHANGES_REQUESTED** [review]

**Comment by bsalamat** [comment]
@marun Please provide your reasoning for moving this PR to scheduling. Otherwise, I will have to revert it.

File: test/e2e/scheduling/ubernetes_lite_volumes.go (hunk 2 above)
Comment (timothysc) [reply test/e2e/scheduling/ubernetes_lite_volumes.go]: The original code mod to enable this feature was in the scheduler.  The key is in the check below - "should schedule pods in the same zones as statically provisioned PVs"

Comment (timothysc) [reply test/e2e/scheduling/ubernetes_lite_volumes.go]: He is moving the ubernetes-lite tests to a proper home.

Comment (bsalamat) [reply test/e2e/scheduling/ubernetes_lite_volumes.go]: The fact that the test has "schedule pods" in the comment is not enough IMO. The question is "does this test mainly tests a scheduling functionality or the functionality of a different component?". I don't see how its purpose is to test a functionality of scheduler.

Comment (timothysc) [reply test/e2e/scheduling/ubernetes_lite_volumes.go]: It's validating this predicate - https://github.com/kubernetes/kubernetes/blob/master/pkg/scheduler/algorithm/predicates/predicates.go#L483

Comment (timothysc) [reply test/e2e/scheduling/ubernetes_lite_volumes.go]: But TBH this test kind of crosses over both storage and scheduling... there isn't really a good home for such things.

**Comment by marun** [comment]
@bsalamat As per @timothysc, the contents of ubernetes_lite_volumes.go appear to be cross-cutting tests whose maintenance should be shared between sig-scheduling and sig-storage.  I've tagged the test accordingly in the new linked PR.  If you feel the tests in question should be the sole responsibility of sig-storage, or some other sig entirely, feel free to raise a PR.  My intent here is to ensure that the tests have one or more sig owners that wasn't sig-multicluster (since multiple clusters are not involved).

File: test/e2e/scheduling/ubernetes_lite_volumes.go (hunk 2 above)
Comment (timothysc) [reply test/e2e/scheduling/ubernetes_lite_volumes.go]: @marun - In chatting with @bsalamat I think we want storage to own these tests.  Even though they are validating predicate behavior, most of the code changes were made by that sig and any issue triage and maintenance should probably be on them.  /cc @kubernetes/sig-storage-misc

Comment (msau42) [reply test/e2e/scheduling/ubernetes_lite_volumes.go]: As the creator of these tests, I am totally fine with owning it.  It's testing the persistent volume admission controller + volume zonal scheduler predicate, and not so much a generic scheduling functionality.

Comment (marun) [reply test/e2e/scheduling/ubernetes_lite_volumes.go]: Would you like to raise a PR to move it to test/e2e/storage or was adding ``[sig-storage]`` back to the description in #58841 sufficient?

Comment (msau42) [reply test/e2e/scheduling/ubernetes_lite_volumes.go]: I think adding it back to the description is fine for now.  I believe this test does use some common functions in the main ubernetes_lite.go file which is why they were put in the same directory.  It'll take some refactoring out of the common parts to put it into the storage directory.
